samples/client/petstore/c/CMakeLists.txt

```diff
@@ -52,34 +52,39 @@ set(HDRS
 
 )
 
+# Add library with project file with projectname as library name
 add_library(${pkgName} SHARED ${SRCS} ${HDRS})
+# Link dependent libraries
 target_link_libraries(${pkgName} ${CURL_LIBRARIES} )
+#install library to destination
 install(TARGETS ${pkgName} DESTINATION ${CMAKE_INSTALL_PREFIX})
 
+# Setting file variables to null
 set(SRCS "")
 set(HDRS "")
 
-# This section shows how to use the above compiled libary to compile the source files
-# set source files
-set(SRCS
-    unit-tests/manual-PetAPI.c
-    unit-tests/manual-StoreAPI.c
-    unit-tests/manual-UserAPI.c
-    unit-tests/manual-order.c
-    unit-tests/manual-user.c)
 
-#set header files
-set(HDRS
-)
+## This section shows how to use the above compiled libary to compile the source files
+## set source files
+#set(SRCS
+#    unit-tests/manual-PetAPI.c
+#    unit-tests/manual-StoreAPI.c
+#    unit-tests/manual-UserAPI.c
+#    unit-tests/manual-order.c
+#    unit-tests/manual-user.c)
+
+##set header files
+#set(HDRS
+#)
```

Comment:
@michelealbano seems like these will disable the integration tests for the C Petstore client. I could no longer find the following in the Travis log:
```
-- Build files have been written to: /home/travis/build/OpenAPITools/openapi-generator/samples/client/petstore/c
Scanning dependencies of target openapi_petstore
[  4%] Building C object CMakeFiles/openapi_petstore.dir/src/list.c.o
[  8%] Building C object CMakeFiles/openapi_petstore.dir/src/apiKey.c.o
[ 12%] Building C object CMakeFiles/openapi_petstore.dir/src/apiClient.c.o
[ 16%] Building C object CMakeFiles/openapi_petstore.dir/external/cJSON.c.o
[ 20%] Building C object CMakeFiles/openapi_petstore.dir/model/api_response.c.o
[ 25%] Building C object CMakeFiles/openapi_petstore.dir/model/category.c.o
[ 29%] Building C object CMakeFiles/openapi_petstore.dir/model/order.c.o
[ 33%] Building C object CMakeFiles/openapi_petstore.dir/model/pet.c.o
[ 37%] Building C object CMakeFiles/openapi_petstore.dir/model/tag.c.o
[ 41%] Building C object CMakeFiles/openapi_petstore.dir/model/user.c.o
[ 45%] Building C object CMakeFiles/openapi_petstore.dir/api/PetAPI.c.o
[ 50%] Building C object CMakeFiles/openapi_petstore.dir/api/StoreAPI.c.o
[ 54%] Building C object CMakeFiles/openapi_petstore.dir/api/UserAPI.c.o
[ 58%] Linking C shared library libopenapi_petstore.so
[ 58%] Built target openapi_petstore
Scanning dependencies of target unit-manual-PetAPI
[ 62%] Building C object CMakeFiles/unit-manual-PetAPI.dir/unit-tests/manual-PetAPI.c.o
[ 66%] Linking C executable unit-manual-PetAPI
[ 66%] Built target unit-manual-PetAPI
Scanning dependencies of target unit-manual-UserAPI
[ 70%] Building C object CMakeFiles/unit-manual-UserAPI.dir/unit-tests/manual-UserAPI.c.o
[ 75%] Linking C executable unit-manual-UserAPI
[ 75%] Built target unit-manual-UserAPI
Scanning dependencies of target unit-manual-StoreAPI
[ 79%] Building C object CMakeFiles/unit-manual-StoreAPI.dir/unit-tests/manual-StoreAPI.c.o
[ 83%] Linking C executable unit-manual-StoreAPI
[ 83%] Built target unit-manual-StoreAPI
Scanning dependencies of target unit-manual-user
[ 87%] Building C object CMakeFiles/unit-manual-user.dir/unit-tests/manual-user.c.o
[ 91%] Linking C executable unit-manual-user
[ 91%] Built target unit-manual-user
Scanning dependencies of target unit-manual-order
[ 95%] Building C object CMakeFiles/unit-manual-order.dir/unit-tests/manual-order.c.o
[100%] Linking C executable unit-manual-order
[100%] Built target unit-manual-order
Data is:{
	"id":	1234,
	"category":	{
		"id":	5,
		"name":	"Example Category"
	},
	"name":	"Example Pet",
	"photoUrls":	["http://www.github.com", "http://www.gitter.im"],
	"tags":	[{
			"id":	1,
			"name":	"beautiful code"
		}, {
			"id":	542353,
			"name":	"at least I tried"
		}],
	"status":	"available"
}
successful operation
Data is:{
	"id":	1234,
	"category":	{
		"id":	5,
		"name":	"Example Category"
	},
	"name":	"Rocky Handsome",
	"photoUrls":	["http://www.github.com", "http://www.gitter.im"],
	"tags":	[{
			"id":	1,
			"name":	"beautiful code"
		}, {
			"id":	542353,
			"name":	"at least I tried"
		}],
	"status":	"sold"
}
successful operation
User is: 
{
	"id":	1234,
	"username":	"example123",
	"firstName":	"Example1",
	"lastName":	"Example2Last",
	"email":	"example@example.com",
	"password":	"thisisexample!123",
	"phone":	"+123456789",
	"userStatus":	4
}
successful operation
Login User: logged in user session:1572762309163
successful operation
Placed order: 
{
	"id":	1234,
	"petId":	12345,
	"quantity":	50,
	"shipDate":	"2018-09-24T10:19:09.592+0000",
	"status":	"placed",
	"complete":	true
}
successful operation
Order received: 
{
	"id":	1234,
	"petId":	12345,
	"quantity":	50,
	"shipDate":	"2018-09-24T10:19:09.592+0000",
	"status":	"placed",
	"complete":	true
}
Order Deleted 
Order not found
successful operation
sold - 2
pending - 2
available - 9
```



Ref: https://travis-ci.org/OpenAPITools/openapi-generator/builds/606108932?utm_source=github_status&utm_medium=notification

Reply:
@wing328 I just run ./bin/c-petstore.sh after the changes by @zhemant 
Should we meet on Slack to decide how it should be?

Reply:
@zhemant I gave a look at the new c-petstore.sh, and I haven't the expertise to correct the issue raised by @wing328 . Can you help with solving it?

Reply:
@michelealbano  Sure,  I think I already have an idea why this is happening. If you can talk on IM, then I can maybe tell you in details about how the working is. I had sent you text on gitter. I don't know if you use it.

Reply:
I've filed https://github.com/OpenAPITools/openapi-generator/pull/4457 with some minor enhancements, which should correct the skipped tests due to regenerating the CMakefile file

Reply:
@michelealbano can you please merge the latest master into this branch as I've merged #4457 into master?